sdk/storage/storage-file-share/src/generated/src/storageClientContext.ts

```diff
@@ -11,7 +11,7 @@
 import * as coreHttp from "@azure/core-http";
 
 const packageName = "azure-storage-file-share";
-const packageVersion = "12.0.0";
+const packageVersion = "1.0.0";
```

Comment:
This should be a TODO, and need a proper version.

Reply:
This version is not used and part of generated code. So do not need to modify it.

Reply:
Minor: Not sure about latest convention here, while changing version for both convenience and generated code before final releasing looks better for me. 

---
In reply to: [357014370](https://github.com/Azure/azure-sdk-for-js/pull/6502#discussion_r357014370) [](ancestors = 357014370)

Reply:
Will fix this when finally releasing.